grammars/less.cson

```diff
@@ -281,12 +281,41 @@
     'name': 'keyword.control.logical.operator'
   }
   {
-    'match': '\\b(a|abbr|acronym|address|applet|area|article|aside|audio|b|base|bdi|bdo|big|blockquote|body|br|button|canvas|caption|center|cite|code|col|colgroup|data|datalist|dd|del|details|dfn|dialog|div|dl|dt|em|embed|eventsource|fieldset|figure|figcaption|footer|form|frame|frameset|(h[1-6])|head|header|hgroup|hr|html|i|iframe|img|input|ins|kbd|keygen|label|legend|li|link|main|map|mark|math|menu|menuitem|meta|meter|nav|noframes|noscript|object|ol|optgroup|option|output|p|param|picture|pre|progress|q|rb|rp|rt|rtc|ruby|s|samp|script|section|select|small|source|span|strike|strong|style|sub|summary|sup|svg|table(?!-)|tbody|td|template|textarea|tfoot|th|thead|time|title|tr|track|tt|u|ul|var|video|wbr)\\b'
-    'name': 'entity.name.tag.html.css'
-  }
-  {
-    'match': '\\b(vkern|view|use|tspan|tref|title|textPath|text|symbol|switch|svg|style|stop|set|script|rect|radialGradient|polyline|polygon|pattern|path|mpath|missing-glyph|metadata|mask|marker|linearGradient|line|image|hkern|glyphRef|glyph|g|foreignObject|font-face-uri|font-face-src|font-face-name|font-face-format|font-face|font|filter|feTurbulence|feTile|feSpotLight|feSpecularLighting|fePointLight|feOffset|feMorphology|feMergeNode|feMerge|feImage|feGaussianBlur|feFuncR|feFuncG|feFuncB|feFuncA|feFlood|feDistantLight|feDisplacementMap|feDiffuseLighting|feConvolveMatrix|feComposite|feComponentTransfer|feColorMatrix|feBlend|ellipse|desc|defs|cursor|color-profile|clipPath|circle|animateTransform|animateMotion|animateColor|animate|altGlyphItem|altGlyphDef|altGlyph|a)\\b'
-    'name': 'entity.name.tag.html.svg.css'
+    'match': '''(?x)
+      (?<![\\w-])
+      (a|abbr|acronym|address|applet|area|article|aside|audio
+      |b|base|bdi|bdo|big|blockquote|body|br|button
+      |canvas|caption|center|cite|code|col|colgroup
+      |data|datalist|dd|del|details|dfn|dialog|div|dl|dt
+      |em|embed|eventsource|fieldset|figure|figcaption|footer|form|frame|frameset
+      |h[1-6]|head|header|hgroup|hr|html|i|iframe|img|input|ins
+      |kbd|keygen|label|legend|li|link|main|map|mark|math|menu|menuitem|meta|meter
+      |nav|noframes|noscript|object|ol|optgroup|option|output
+      |p|param|picture|pre|progress|q|rb|rp|rt|rtc|ruby
+      |s|samp|script|section|select|small|source|span|strike|strong|style|sub|summary|sup|svg
+      |table|tbody|td|template|textarea|tfoot|th|thead|time|title|tr|track|tt
+      |u|ul|var|video|wbr)
```

Comment:
You could remove acronym, applet, big, center, eventsource, frame, frameset, noframes, strike, and tt.